HousingSearchListener/HousingSearchListener.cs

```diff
@@ -84,6 +88,21 @@ private async Task ProcessMessageAsync(SQSEvent.SQSMessage message, ILambdaConte
                                 processor = ServiceProvider.GetService<IIndexTenureUseCase>();
                                 break;
                             }
+                        case EventTypes.AccountCreatedEvent:
```

Comment:
At this point (as this seems to grow), I would move it to its own factory class, and get the processor type from there. (I mean the entire case statement)

Reply:
Yea, I aggree. Will change it